### PR TITLE
Fix report

### PR DESCRIPTION
**Pull request by RutsSantos**
#### Description:
Fix the Report color and icons. Add a World countries picker if the user have traveled.

#### Linked issues:
[Ticket #74](https://trello.com/c/e7ZZ2WCJ/74-fix-report-visuals-and-icons)

#### Screenshots:
Simulator Screen Shot - iPhone 11 - 2020-05-22 at 10 57 09
Simulator Screen Shot - iPhone 11 - 2020-05-22 at 10 57 50

### CODE REVIEW

File: ios/Podfile.lock [hunk 1]
```diff
@@ -5,14 +5,14 @@ PODS:
   - BVLinearGradient (2.5.6):
     - React
   - DoubleConversion (1.1.6)
-  - FBLazyVector (0.62.2)
```
Comment (adnanjt) [inline ios/Podfile.lock:8]: Podemos obviar estos cambios ?

File: ios/Podfile [hunk 2]
```diff
@@ -47,7 +47,6 @@ target 'COVIDSafePaths' do
   pod 'RNReanimated', :path => '../node_modules/react-native-reanimated'
 
 
-  pod 'react-native-maps', :path => '../node_modules/react-native-maps'
```
Comment (adnanjt) [inline ios/Podfile:50]: Este es necesario ?

File: ios/COVIDSafePaths.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -595,6 +596,54 @@
 			shellPath = /bin/sh;
 			shellScript = "export NODE_BINARY=node\n../node_modules/react-native/scripts/react-native-xcode.sh\n";
 		};
+		977C89BE482C6DB44196845B /* [CP] Copy Pods Resources */ = {
```
Comment (adnanjt) [inline ios/COVIDSafePaths.xcodeproj/project.pbxproj:599]: Podemos obviar estos cambios ?

File: ios/Podfile.lock (hunk 1 above)
Comment (RutsSantos) [reply ios/Podfile.lock:8]: Dices quitar el podfile.lock ?

**Review by adnanjt — COMMENTED** [review]
Other than thos questions LGTM, answer to approve

File: ios/COVIDSafePaths.xcodeproj/project.pbxproj (hunk 3 above)
Comment (RutsSantos) [reply ios/COVIDSafePaths.xcodeproj/project.pbxproj:599]: No veo porqué no

File: ios/Podfile (hunk 2 above)
Comment (adnanjt) [reply ios/Podfile:50]: vamos a dejarlo

File: ios/COVIDSafePaths.xcodeproj/project.pbxproj (hunk 3 above)
Comment (adnanjt) [reply ios/COVIDSafePaths.xcodeproj/project.pbxproj:599]: Never mind

**Review by inteljudajocu — APPROVED** [review]
Fuera de los comments de @adnanjt, LGTM

**Review by JoseFcoEstrella — APPROVED** [review]
LGTM

**Review by adnanjt — APPROVED** [review]
Ready please pull after merge to test the environments

**Pull request merged**